main/2024/WG-03-26.md

```diff
@@ -0,0 +1,40 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the 2024-03-26 video call of WebAssembly's Working Group
```

Comment:
This (and the name of the file) should be CG, not WG.

Actually it looks like you copied the template from a WG meeting rather than a CG meeting.

Reply:
I just updated the template for CG meetings at https://github.com/WebAssembly/meetings/blob/main/main/CG-template.md you can use that

Reply:
I raced with you and upated this PR based on a previous meeting

Reply:
No problem, using a previous meeting is fine.